boba/cli.py

```diff
@@ -48,14 +52,46 @@ def print_help(err=''):
     ctx.exit()
 
 
+def run_batch_of_universes(folder, universes):
+    batch = []
+    for universe in universes:
+        batch.append(run_universe(folder, universe))
+
+    return batch
+
+
+def run_universe(folder, script):
+    out = None
+    # choose python or R based on file extension of provided universe
+    if(script.split('.')[1] == "py"):
```

Comment:
Code style fix: do not mix single and double quotes! In this project, we use single quotes for strings whenever possible.

Reply:
We have a dedicated file for handling file extension, language, etc. It is here: https://github.com/uwdata/multiverse-spec/blob/master/boba/lang.py

Let's use `lang.py` for all such operations, so it's easier to extend the support for other languages in the future.